mapper/pacs_008_transform.bal

```diff
@@ -0,0 +1,573 @@
+    SwiftMtRecords:MT57B? MT57B = accountWithInstitution is SwiftMtRecords:MT57B ? check accountWithInstitution.ensureType(SwiftMtRecords:MT57B) : ();
+    SwiftMtRecords:MT57C? MT57C = accountWithInstitution is SwiftMtRecords:MT57C ? check accountWithInstitution.ensureType(SwiftMtRecords:MT57C) : ();
+
+    SwiftMtRecords:MT59? | SwiftMtRecords:MT59A? | SwiftMtRecords:MT59F? beneficiaryCustomer = getMT103BeneficiaryCustomerFromPacs008Document(document);
+    SwiftMtRecords:MT59? MT59 = beneficiaryCustomer is SwiftMtRecords:MT59 ? check beneficiaryCustomer.ensureType(SwiftMtRecords:MT59) : ();
+    SwiftMtRecords:MT59A? MT59A = beneficiaryCustomer is SwiftMtRecords:MT59A ? check beneficiaryCustomer.ensureType(SwiftMtRecords:MT59A) : ();
+    SwiftMtRecords:MT59F? MT59F = beneficiaryCustomer is SwiftMtRecords:MT59F ? check beneficiaryCustomer.ensureType(SwiftMtRecords:MT59F) : ();
+
+    SwiftMtRecords:MT70 MT70 = getRemitenceInformationFromPmtIdOrRmtInf(firstTransaction.PmtId, firstTransaction.RmtInf);
+
+    SwiftMtRecords:MT71A MT71A = {
+        name: "71A",
+        Cd: getDetailsOfChargesFromChargeBearerType1Code(firstTransaction.ChrgBr)
+    };
+
+    SwiftMtRecords:MT71F MT71F = check convertCharges16toMT71F(firstTransaction.ChrgsInf);
+
+    SwiftMtRecords:MT71G MT71G = check convertCharges16toMT71G(firstTransaction.ChrgsInf);
+
+    SwiftMtRecords:MT72 MT72 = { name: "72", Cd: { \#content: "", number: "1" } };
+
+    SwiftMtRecords:MT77B MT77B = {
+        name: "77B",
+        Nrtv: { \#content: "", number: "1" }
+    };
+
+    SwiftMtRecords:MT77T MT77T = {
+        name: "77T",
+        EnvCntnt: {\#content: "", number: ""}
+    };
+
+    match messageType {
+        MT103 => {
+            return <SwiftMtRecords:MT103Block4> {
+                MT20, MT13C, MT23B, MT23E, MT26T, MT32A, MT33B, MT36, MT50A, MT50F, MT50K, MT51A, MT52A, MT52D, 
+                MT53A, MT53B, MT54A, MT54B, MT54D, MT55A, MT55B, MT55D, MT56A, MT56C, MT56D, MT57A, MT57B, MT57C, MT59, MT59A, MT59F,  
+                MT70, MT71A, MT71F, MT71G, MT72, MT77B
+            };
+        }
+
+        MT103_STP => {
+            return <SwiftMtRecords:MT103STPBlock4> {
+                MT20, MT13C, MT23B, MT23E, MT26T, MT32A, MT33B, MT36, MT50A, MT50F, MT50K, MT52A, MT53A, MT53B,
+                MT54A, MT55A, MT56A, MT57A, MT59, MT59A, MT59F, MT70, MT71A, MT71F, MT71G, MT72, MT77B
+            };
+        }
+
+        MT103_REMIT => {
+            return <SwiftMtRecords:MT103REMITBlock4> {
+                MT20, MT13C, MT23B, MT23E, MT26T, MT32A, MT33B, MT36, MT50A, MT50F, MT50K, MT51A, MT52A, MT52D, 
+                MT53A, MT53B, MT54A, MT54B, MT54D, MT55A, MT55B, MT55D, MT56A, MT56C, MT56D, MT57A, MT57B, MT57C, MT59, MT59A, MT59F, MT71A, 
+                MT71F, MT71G, MT72, MT77B, MT77T
+            };
+        }
+    }
+
+    return error("Error occurred while creating MT103 block4");
+
+}
```

Comment:
Add EoF line for all the files

Reply:
Done in https://github.com/ballerina-platform/module-ballerinax-financial.iso20022-to-swiftmt/pull/1/commits/15280bae5517d16c94776c14140deae3301141ca